codersdk/templates.go

```diff
@@ -232,7 +232,7 @@ type UpdateTemplateMeta struct {
 	// RequireActiveVersion mandates workspaces built using this template
 	// use the active version of the template. This option has no
 	// effect on template admins.
-	RequireActiveVersion bool `json:"require_active_version"`
+	RequireActiveVersion bool `json:"require_active_version,omitempty"`
```

Comment:
I tested this a few times, including the e2e test, and it looks like it works ok by just adding `omitempty`.

Reply:
Yeah it seems setting `omitempty` generates an optional type for the typescript payload. So you can omit the value in the scheduling form and it seems to set it to its existing value from the template.